src/main/java/edu/umich/kanban/XmlMerge.java

```diff
@@ -44,7 +44,7 @@ public String mergeFiles(String queryString, String xsltPath, String xmlPath) th
       
       String its=queryJiraITS(queryString);
       String sakai = queryJiraSakai(queryString);
-      String internet2 = queryJiraInternet2(queryString);
+      //String internet2 = queryJiraInternet2(queryString);
```

Comment:
Would prefer just deleting this rather than commented out code (same with following lines)